database.sql

```diff
@@ -0,0 +1,10 @@
+CREATE DATABASE IF NOT EXISTS Notes;
+
+CREATE TABLE IF NOT EXISTS Notepad
+(
+    ID              int             NOT NULL        PRIMARY KEY AUTO_INCREMENT,
+    Note_authors    varchar (70)    DEFAULT NULL,
+    Note_titles     varchar (255)   DEFAULT NOT NULL,
```

Comment:
255 is maybe lil to big for a title

Reply:
For me this is only the maximum length that is allowed. If someone -really- wants to make a title with 255 they can. But 256 will be too long.